src/LessMsi.Gui/MainForm.cs

```diff
@@ -914,7 +928,7 @@ private void btnExtract_Click(object sender, EventArgs e)
 			var selectedFiles = new List<MsiFile>();
 			if (fileGrid.SelectedRows.Count == 0)
 			{
-				ShowUserMessageBox("Please select some or all of the files to extract them.");
+				ShowUserMessageBox($"{Strings.SelectFilesMessage}.");

```

Comment:
```suggestion
				ShowUserMessageBox(Strings.SelectFilesMessage);

```
Punctuation (in this case the period) should be in the translation itself.